concurrent.lib.sh

```diff
@@ -315,6 +315,7 @@ concurrent() (
             while read -r meta; do
                 printf "meta:%d:%s\n" "${1}" "${meta}" >> "${4}"
             done
+            exit ${PIPESTATUS[0]}
```

Comment:
One final thing! Shellcheck is still unhappy with the line, although it's actually wrong in this case.

```
In concurrent.lib.sh line 318:
            exit ${PIPESTATUS[0]}
                 ^-- SC2086: Double quote to prevent globbing and word splitting.
```

However, since I'd like the Travis build to pass, I see two acceptable options here:

Either quote the line...

```bash
exit "${PIPESTATUS[0]}"
```

...or disable this specific check for this line.

```bash
# shellcheck disable=SC2086
exit ${PIPESTATUS[0]}
```